core/src/main/scala/cats/std/anyval.scala

```diff
@@ -15,129 +14,103 @@ trait AnyValInstances
   with    BooleanInstances
   with    UnitInstances
 
-trait IntInstances extends algebra.std.IntInstances {
+trait IntInstances extends cats.kernel.std.IntInstances {
 
   implicit val intShow: Show[Int] =
     Show.fromToString[Int]
 
   implicit val intGroup: CommutativeGroup[Int] =
-    AdditiveCommutativeGroup[Int].additive
-
+    new CommutativeGroup[Int] {
```

Comment:
I was going to ask you about that -- we didn't have generic semigroups for numbers in Algebra. If everyone agrees that we want those I can move them in this PR.